README.md

```diff
@@ -99,6 +99,18 @@ When a value is set specifically for Twitter, it will be prioritized over the ge
 seo()->twitterTitle('About us')
 ```
 
+### Favicons
+
+By default, no favicon links will be included. You can manually enable the extension by calling:
+
+```php
+seo()->favicon('path/to/logo.png');
+```
+
+We'll generate a 32x32px `favicon.ico` & `favicon.png` icon. This should be sufficient for most cases.
```

Comment:
Add a note saying that this will create `public/favicon.ico` and `public/favicon.png` so that it's clear that the files will be stored there (in case someone already has other favicons there and wasn't expecting this to override them).